recipes/spleeter-gpu/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "spleeter-gpu" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 49e776f26c561357eb9608a459030ab46cfe11edf21fb17470630a0d3777d453
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [osx or py<36 or py>37]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - requests
+    - setuptools
+    - importlib_resources  # [py<37]
+    - tensorflow-gpu ==1.14.0
+    - pandas ==0.25.1
```

Do this need to be pinned so tightly? As it stands it will fail to build with Python 3.8 as only newer patch versions are available
```suggestion
    - pandas >=0.25
```

---

Yes, we does not support Python 3.8 at the moment. We will update the library in a future iteration for Python 3.8 support (and we will upgrade all dependencies as well, especially Tensorflow which will allow to release all the dependencies version constraints ).